scanapp/models.py

```diff
@@ -31,86 +31,55 @@
 from rest_framework.authtoken.models import Token
 
 
-# Create your models here.
 @receiver(post_save, sender=settings.AUTH_USER_MODEL)
 def create_auth_token(sender, instance=None, created=False, **kwargs):
     if created:
         Token.objects.create(user=instance)
 
 
-class ScanInfo(models.Model):
+class Scan(models.Model):
+    """
+    Store various attributes of a scan like `scan_type`, `is_complete`, `user`, 
+    `URL`, `code_directory`, `total_code_files`, `code_size`, `scancode_notice`,
+    `scancode_version`, `files_count`, `total_error`, `scan_time`
+    """
     def __str__(self):
         return self.scan_type
 
-    # types of scans that can be applied
     scan_types = (
-        ('URL', 'URL'),
-        ('Local Scan', 'localscan'),
+        ('url', 'url'),
```

Comment:
This can be done afterwards but we had said that we would use only one scan input, not multiple types